src/Plutus/Types/CurrencySymbol.js

```diff
@@ -0,0 +1,13 @@
+/* global require exports BROWSER_RUNTIME */
+
+var CardanoWasm;
+if (typeof BROWSER_RUNTIME != 'undefined' && BROWSER_RUNTIME) {
+    CardanoWasm = require('@ngua/cardano-serialization-lib-browser');
+} else {
+    CardanoWasm = require('@ngua/cardano-serialization-lib-nodejs');
+}
+
+exports._unsafeScriptHashFromBytes = input => {
```

Comment:
I have my doubts about introducing this module since a change can be out of sync with the definitions on `src/Serialization/Hash.js`. I introduced this module as I didn't want to export `_unsafeScriptHashFromByte` or its variants from the module `Serialization.Hash` to avoid its abuse.

Reply:
Use `Deserialization.FromBytes` typeclass. Add a new instance for `ScriptHash`. This whole module can be removed.

Reply:
In the end, I deleted it. 

We can't use the `FromBytes` module as we would get an `E r CurrencySymbol` and we want just a plain `CurrencySymbol`, we can get rid of the possibility of error in the signature as the smart constructor for `CurrencySymbol` should already do the checks.